go.mod

```diff
@@ -6,6 +6,88 @@ require (
 	github.com/fxamacker/cbor/v2 v2.4.0
 	github.com/golang-jwt/jwt/v4 v4.5.0
 	github.com/veraison/go-cose v1.0.0
+	golang.org/x/crypto v0.0.0-20220824171710-5757bc0c5503
+	github.com/cloudflare/cfssl v1.6.3
```

Comment:
Is the use of `github.com/cloudflare/cfssl` approved by crypto board of any organization?